nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -16,9 +16,13 @@ import "zeppelin/token/ERC20/SafeERC20.sol";
 */
 interface PolicyManagerInterface {
     function register(address _node, uint16 _period) external;
-    function updateFee(address _node, uint16 _period) external;
     function escrow() external view returns (address);
-    function setDefaultFeeDelta(address _node, uint16 _period) external;
+    function ping(
+        address _node,
+        uint16 _processedPeriod1,
+        uint16 _processedPeriod2,
```

Comment:
What do you think of a more descriptive name like `_currentCommittedPeriod` and `_nextCommittedPeriod`?

Reply:
Hmm, both periods probably will be in a past, `current` and `next`  could confuse, what do you think?

Reply:
`_from` and `_to`?

Reply:
it's not `_from` and `_to`) tese periods woth comitted some time in a past and now minted/processed

Reply:
What about `_last[Processed/Minted]CurrentCommittedPeriod` and `_last[Processed/Minted]NextCommittedPeriod`?...kinda wordy but something more descriptive